example/README.md

```diff
@@ -1,88 +1,223 @@
-even when a user closes your app. 
+* see a [Flutter example app](flutter/objectbox_demo)
 
-If you didn't specify this path to ObjectBox, it would try to use a default "objectbox" directory where the app is 
-currently running, but it doesn't have permissions to write there: `failed to create store: 10199 Dir does not exist: objectbox (30)`.
+Data model
+--------------------
+
+In the following file, e.g. `models.dart`, we import ObjectBox to access annotations.
+Then, we define a single "Entity" that should be persisted by ObjectBox. You can have multiple
+entities in the same file, or you can have them spread across multiple files in your package's `lib` directory.
+
+```dart
+import 'package:objectbox/objectbox.dart';
+
+@Entity()
+class Note {
+    // Each "Entity" needs a unique integer ID property.
+    // Add `Id()` annotation if its name isn't "id" (case insensitive).
+    int id;
+
+    String text;
+
+    int date;
+
+    @Transient() // Make this field ignored, not stored in the database.
+    int notPersisted;
+
+    // An empty default constructor is needed but you can use optional args.
+    Note({this.text});
+
+    // Note: just for logs in the examples below(), not needed by ObjectBox.
+    toString() => 'Note{id: $id, text: $text}';
+}
+```
+
+To generate ObjectBox binding code for your entities, run `(flutter|dart) pub build_runner build`.
+ObjectBox generator will look for all `@Entity` annotations in your `lib` folder and create a single database definition
+`lib/objectbox-model.json` and supporting code in `lib/objectbox.g.dart`.
+You should commit `objectbox-model.json` into your source control (e.g. git) and add `objectbox.g.dart` to the ignore
+list (e.g. .gitignore), otherwise the build_runner will complain about it being changed each time you pull a change.
+
+> The generator will process `lib` and `test` folders separately and create a separate database in each one, if it finds
+> annotations there. This is useful if you need a separate test DB. If you're just writing tests for your own code, you
+> won't have any annotations in the `test` folder so no DB will be created there.
+
+Creating a store
+----------------
+
+`Store` is your entrypoint to ObjectBox - see below how to open it, based on the SDK you're using.
+
+### Flutter apps
+
+On mobile devices, you should store data in your app documents directory - it stays there even when you close the app.
+
+Use `getApplicationDocumentsDirectory()` from the `path_provider` package to retrieve this directory.
 
-To configure ObjectBox properly, you can use `getApplicationDocumentsDirectory()` from the `path_provider` package. 
-See [Flutter: read & write files](https://flutter.dev/docs/cookbook/persistence/reading-writing-files) for more info. 
-Have a look how it's done in the Flutter example app:
 ```dart
 import 'package:path_provider/path_provider.dart';
 
+import 'objectbox.g.dart'; // created by `pub run build_runner build`
```

Comment:
Hm, above the command was `(flutter|dart) pub build_runner build`, am I missing something?

Reply:
```suggestion
import 'objectbox.g.dart'; // created by `flutter pub run build_runner build`
```